shared_model/interfaces/permissions.hpp

```diff
@@ -100,11 +112,6 @@ namespace shared_model {
       PermissionSet<Perm> &operator&=(const PermissionSet<Perm> &r);
       PermissionSet<Perm> &operator|=(const PermissionSet<Perm> &r);
       PermissionSet<Perm> &operator^=(const PermissionSet<Perm> &r);
-
-     private:
-      constexpr auto bit(Perm p) const {
-        return static_cast<size_t>(p);
-      }
     };
 
     extern template class PermissionSet<permissions::Role>;
```

Comment:
This lines produce warnings in Swig (320) a probably redundant here and can be removed safely, what do you think?

Reply:
Yes it causes warning, though cannot be removed:
1. it's **significantly** optimize template instantiations (once, not at each include)
2. it cannot be removed, just because template must be instantiated somewhere (definitions from cpp file must be moved here)